tasks/eth/fp-recovery/001-blacklist-game/README.md

```diff
@@ -0,0 +1,65 @@
+# Deputy Guardian - Blacklist Dispute Game
+
+Status: CONTINGENCY TASK, SIGN AS NEEDED
+
+## Objective
+
+This task executes a `blacklistDisputeGame` operation in the `OptimismPortalProxy` on behalf of the `Guardian` role. Blacklisting the dispute game prevents any withdrawals proven to be contained within the game's proposed output root from being finalized.
+
+The batch will be executed on chain ID `1`, and contains `1` transaction.
+
+## Tx #1: Blacklist Dispute Game
+
+Executes the `blacklistDisputeGame` call to the `OptimismPortalProxy`.
+
+**Function Signature:** `blacklistDisputeGame(address,address)`
+
+**To:** `0x4220C5deD9dC2C8a8366e684B098094790C72d3c`
+
+**Value:** `0 WEI`
+
+**Raw Input Data:** `0x629cdd49000000000000000000000000<OptimismPortalProxyAddress------------>000000000000000000000000<DisputeGameToBlacklist---------------->`
+
+### Inputs
+
+**\_portal:** `<user-input>`
+
+**\_game:** `<user-input>`
+
+## Preparing the Operation
+
+1. Locate the address of the `OptimismPortalProxy` to blacklist a dispute game on.
```

Comment:
NITS:  I think adding some information how to find the address of the `OptimismPortalProxy` using the Superchain Registry is also worth it here let me know ( https://github.com/ethereum-optimism/superchain-registry/blob/2c96a89df841013a59269fa7adc12c77b870310e/superchain/configs/mainnet/base.toml#L52)

Reply:
Agreed, updated the wording here to be more descriptive. lmk if you want any additional changes here